### DISCUSSION TITLE
Unique Indexes mongodb with FastAPI

### DISCUSSION THREAD

**Discussion by sonpxp** [fastapi/fastapi#4938] [Questions]
how to create unique index in mongodb with FastAPI

**Comment by nikstuckenbrock** [comment]
Hey @sonpxp,

you can use the `create_index` function for that.

```python
import pymongo
mongo_client['collection'].create_index(['property_name': pymongo.ASCENDING)], unique=True)
```

If you need more than one index you can simply add more properties to the array as described [here](https://stackoverflow.com/questions/35812685/pymongo-create-unique-index-with-2-or-more-fields#35812898).
For further information just look at the [pymongo documentation](https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.create_index).

I hope I've been able to help?
